app/controllers/application_controller.rb

```diff
@@ -111,6 +111,8 @@ def not_authorized
   end
 
   def bad_request
+    raise unless request.format.json?
```

Comment:
This is a catch-all `rescue_from ArgumentError`, which is _really_ nice to have in the API response, but obscures the "real error" when testing in development-mode.